### ISSUE TITLE
Improved icons for macOS

### ISSUE THREAD

**Issue by cameronwhite** [PintaProject/Pinta#908]
From https://github.com/PintaProject/Pinta/discussions/892, there are some improved icons that have been contributed. For macOS (and maybe Windows) we could ship these, replacing the default icons bundled with Pinta. This would need a bit of work in the build system scripts to set this up.
We shouldn't do this for Linux since the system GTK icon theme is being used

**Comment by pedropaulosuzuki** [comment]
I don't like the idea of platform specific icons. Maybe we could ship the two icon themes and then create a setting to switch between them.

However, I believe the icons suggested don't account for all of the icons we use in Pinta (for example, `palette-symbolic`, all of `join-symbolic`, all of `font-weight`), which would mean we would have to recreate those following this style or risk having different icon styles all around, which is also really bad.

**Comment by cameronwhite** [comment]
Yeah, I'm less sure about this idea after revisiting it. Now that we have more custom icons, which are trying to be a similar style to the default adwaita icons, shipping alternate versions of only the standard icons doesn't seem worthwhile.

**Comment by KevDoy** [comment]
I say we close this then.

**Comment by pedropaulosuzuki** [comment]
> I say we close this then.

You're the creator of [Ysui](https://ysui.org), right? If you think these additional icons (like #1981, #1986, #2008 and #2052, for example) are a good addition to your icon pack, then we could make them an option for Pinta as well. I just disagree with those icons being exclusive to a specific OS, but I'm not against them being an option for Pinta. Though we would probably bother you when we add new icons, so it's up to you if you think that's worthwhile or not.

**Comment by KevDoy** [comment]
ysui has been on the backburner right now, but I'm more than happy to jump back in when icons are needed! I'll take a look at these.
